articles/quickstart/native/ios-objc/_includes/_login_centralized.md

```diff
@@ -114,6 +114,27 @@ To learn more about the `credentials` object, read the [Credentials](https://git
 
 <%= include('../../../../_includes/_logout_url') %>
 
+To clear on server side the session add the following snipped to the HybridAuth.swift file:
+
+```swift
+// HybridAuth.swift
+
+    @objc
```

Comment:
See if you can remove all these spaces on the left so that the **starting** indentation matches the `// HybridAuth.swift` line and so, all the snippets above.